opentelemetry-sdk/src/opentelemetry/sdk/util.py

```diff
@@ -28,3 +39,105 @@ def ns_to_iso_str(nanoseconds):
     """Get an ISO 8601 string from time_ns value."""
     ts = datetime.datetime.fromtimestamp(nanoseconds / 1e9)
     return ts.strftime("%Y-%m-%dT%H:%M:%S.%fZ")
+
+
+class BoundedList(Sequence):
```

Comment:
Why move these in this PR if you're not using them?

Reply:
Currently, the `Meter` implementation has no concept of limits on certain data objects (`handle`, `metric`), but could possibly have those limitations in the future, in which it can leverage these data structures. They also seem reusable enough to put in `utils.py`. I remember there was a [comment](https://github.com/open-telemetry/opentelemetry-python/pull/58#discussion_r305585111) about moving it to a common place. I don't have a strong opinion though.